numpy/__init__.py

```diff
@@ -185,8 +185,12 @@ def pkgload(*packages, **options):
     pkgload.__doc__ = PackageLoader.__call__.__doc__
 
     from .testing import Tester
-    test = Tester().test
-    bench = Tester().bench
+    if '.dev0' in __version__:
+        test = Tester(raise_warnings="develop").test
+        bench = Tester(raise_warnings="develop").bench
+    else:
+        test = Tester(raise_warnings="release").test
+        bench = Tester(raise_warnings="release").bench
```

Comment:
Ahh sorry, didn't notice the location.

FYI if line count is at a premium, my proposed 4-line conditional can actually fit on a single line. Add in the other three lines plus the requisite new single del line yields 5 total lines to do the same thing instead of the current six (with the advantage of using a single instance). But it probably doesn't matter much either way...